settings.gradle

```
@@ -6,7 +6,7 @@ pluginManagement {
 }
 
 plugins {
-	id "com.gradle.enterprise" version "3.7.2"
+	id "com.gradle.enterprise" version "3.8.1"
 	id "io.spring.ge.conventions" version "0.0.7"
```

```suggestion
	id "io.spring.ge.conventions" version "0.0.9"
```

See https://github.com/spring-io/gradle-enterprise-conventions/releases/tag/v0.0.9

---

oh, my bad, I got confused with Github's PR view.
I thought the suggestion was to remove the declaration of `com.gradle.enterprise` plugin

This is definitely a valid suggestion 😅